<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trabalho 7 - CPP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="imagePanel">
      <property name="minimumSize">
       <size>
        <width>538</width>
        <height>98</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/MyImages/logo.png);</string>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="inputPanel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>238</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Participantes</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="field">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>151</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="startButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>40</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Iniciar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="resetButton">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>40</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblWinner">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>471</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>8ยบ Ganhador</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="viewerPanel">
      <property name="title">
       <string>View</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="winnerInView">
         <property name="font">
          <font>
           <pointsize>48</pointsize>
          </font>
         </property>
         <property name="text">
          <string>32</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sortButton">
      <property name="text">
       <string>Sortear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>556</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
